src/oci-config.c

```diff
@@ -159,3 +159,48 @@ cc_oci_config_free (struct cc_oci_config *config)
                 (GDestroyNotify)cc_oci_ns_free);
 	}
 }
+
+/*!
+ * find and call the spec handler for each child of GNode
+ *
+ * \param [in] node GNode
+ * \param[in,out] config cc_oci_config struct
+ *
+ * \return \c false if a spec handler fails, else \c true
+ */
+gboolean
+cc_oci_process_config (GNode* root, struct cc_oci_config* config,
```

Comment:
The expected code style is `struct foo *a` and not `struct foo* a`. For having a consistent format across the code, I'd appreciate if you could fix this.